examples/cryptocurrency-advanced/backend/src/api.rs

```diff
@@ -22,16 +22,16 @@ use exonum::{
 
 use transactions::WalletTransactions;
 use wallet::Wallet;
-use {CurrencySchema, CRYPTOCURRENCY_SERVICE_ID};
+use {Schema, CRYPTOCURRENCY_SERVICE_ID};
 
-/// The structure describes the query parameters for the `get_wallet` endpoint.
+/// Describes the query parameters for the `get_wallet` endpoint.
 #[derive(Debug, Clone, Copy, Serialize, Deserialize, PartialEq)]
 pub struct WalletQuery {
     /// Public key of the queried wallet.
     pub pub_key: PublicKey,
 }
 
-/// The structure returned by the REST API.
+/// Response to a sent transaction returned by the REST API.
```

Comment:
Is it a transaction received by the service? If so, then it could be rephrased to 'Response to an incoming transaction'.